### PR TITLE
Editor namespaces refactoring - part 1

### PR DESCRIPTION
**Pull request by tmat**
Match namespaces to top-level directory names in EditorFeatures project.

### CODE REVIEW

File: src/EditorFeatures/Core/AddImports/AbstractAddImportsPasteCommandHandler.cs [hunk 1]
```diff
@@ -18,7 +16,7 @@
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor.Commanding.Commands;
 
-namespace Microsoft.CodeAnalysis.Editor.Implementation.AddImports
+namespace Microsoft.CodeAnalysis.AddImport
```
Comment (dibarbet) [inline src/EditorFeatures/Core/AddImports/AbstractAddImportsPasteCommandHandler.cs:19]:
```suggestion
namespace Microsoft.CodeAnalysis.AddImports
```

File: src/EditorFeatures/Core/BraceMatching/BraceHighlightTag.cs [hunk 2]
```diff
@@ -4,9 +4,10 @@
 
 #nullable disable
 
+using Microsoft;
```
Comment (dibarbet) [inline src/EditorFeatures/Core/BraceMatching/BraceHighlightTag.cs:7]: what is in this namespace?

File: src/EditorFeatures/Core/CommentSelection/AbstractToggleBlockCommentBase.cs [hunk 3]
```diff
@@ -10,7 +10,8 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.CommentSelection;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Editor.Implementation.CommentSelection;
```
Comment (dibarbet) [inline src/EditorFeatures/Core/CommentSelection/AbstractToggleBlockCommentBase.cs:14]: should the items in this namespace be `Microsoft.CodeAnalysis.CommentSelection`? (and below)

File: src/EditorFeatures/Core/CommentSelection/CommentSelectionResult.cs [hunk 4]
```diff
@@ -6,6 +6,7 @@
 
 using System.Collections.Generic;
 using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CommentSelection;
 using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.CommentSelection
```
Comment (dibarbet) [inline src/EditorFeatures/Core/CommentSelection/CommentSelectionResult.cs:12]: should this be `Microsoft.CodeAnalysis.CommentSelection`?

**Review by dibarbet — APPROVED** [review]

File: src/EditorFeatures/Core/AddImports/AbstractAddImportsPasteCommandHandler.cs (hunk 1 above)
Comment (tmat) [reply src/EditorFeatures/Core/AddImports/AbstractAddImportsPasteCommandHandler.cs:19]: This is intentional. We already have AddImport namespace in Features.

File: src/EditorFeatures/Core/BraceMatching/BraceHighlightTag.cs (hunk 2 above)
Comment (tmat) [reply src/EditorFeatures/Core/BraceMatching/BraceHighlightTag.cs:7]: Shouldn't have been added. Bug in refactoring: https://github.com/dotnet/roslyn/issues/59899

**Pull request merged**